week03/code/SetsAndMaps_Tests.cs

```diff
@@ -184,22 +184,23 @@ public void IsAnagram_IgnoresSpacesAndCases()
     }
 
     // If this test takes longer than 5 seconds to run, your code is too inefficient.
-    //  On my machine, this executes in ~1 second with an efficient implementation.
+    //  On my machine, this executes in ~3 seconds with an efficient implementation.
     [TestMethod, Timeout(5000)]
     public void IsAnagram_Efficiency()
     {
         var rand = new Random();
-        var length = 30_000_000;
+        var length = 60_000_000;
```

Comment:
Update constants to provide more room between correct and incorrect implementations.